### ISSUE TITLE
 miopen only supports 4D tensors, dim=5 not allowed

### ISSUE THREAD

**Issue by guojunxia** [ROCmSoftwarePlatform/tensorflow-upstream#1519]
"tensorflow/stream_executor/rocm/rocm_dnn.cc:477] miopen only supports 4D tensors, dim=5 not allowed"
hi, this error happens when I use tf.layers.conv3d_transpose in rocm2.9_tensorflow1.13_python3.7 .
Has anyone encountered it before?
Do i need to updata my rocm or miopen?
I'm also confused my  tf.layers.conv3d() also has 5 parameters but it passes QAQ

**Comment by guojunxia** [comment]
This issued has been solved when i use  compiler/dtk/21.04 even though i don't know why  it works.
